<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentationVerification</class>
 <widget class="qMRMLWidget" name="SegmentationVerification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>480</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_view_number">
     <item>
      <widget class="QLabel" name="label_view_number">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Number of Views:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="viewspinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_view_names">
     <item>
      <widget class="QLabel" name="label_view_names">
       <property name="text">
        <string>Insert View Names:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="viewNames"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_view">
     <item>
      <widget class="QCheckBox" name="threedCheckbox">
       <property name="text">
        <string>Enable 3D View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="twodCheckbox">
       <property name="text">
        <string>Enable 2D View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Segmentation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="segmentationNodeComboBox">
       <property name="nodeTypes">
        <stringlist notr="true">
         <string>vtkMRMLSegmentationNode</string>
        </stringlist>
       </property>
       <property name="hideChildNodeTypes">
        <stringlist notr="true"/>
       </property>
       <property name="interactionNodeSingletonTag">
        <string notr="true"/>
       </property>
       <property name="noneDisplay">
        <string>Select a segmentation to verify...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Show neighboring segments semi-transparent:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showNeighborsCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="qMRMLSegmentsTableView" name="SegmentsTableView">
     <property name="selectionMode">
      <number>1</number>
     </property>
     <property name="statusColumnVisible">
      <bool>false</bool>
     </property>
     <property name="filterBarVisible">
      <bool>true</bool>
     </property>
     <property name="textFilter">
      <string/>
     </property>
     <property name="jumpToSelectedSegmentEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="previousButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Run the algorithm.</string>
       </property>
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Run the algorithm.</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentsTableView</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentsTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>SegmentationVerification</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segmentationNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>3</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SegmentationVerification</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SegmentsTableView</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>2</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>segmentationNodeComboBox</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>SegmentsTableView</receiver>
   <slot>setSegmentationNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
